.github/workflows/ci.yml

```diff
@@ -101,6 +100,7 @@ jobs:
             apt:
               - libopenjp2-7
               - graphviz
+              - libcairo2-dev
```

Comment:
Why is this needed?

Reply:
Good question, I needed it on 6.0 branch, as at the time cario was being compiled so I needed it.